Mathlib/Analysis/ODE/PicardLindelof.lean

```diff
@@ -304,8 +304,7 @@ theorem dist_next_apply_le_of_le {f₁ f₂ : FunSpace v} {n : ℕ} {d : ℝ}
       refine norm_integral_le_of_norm_le (Continuous.integrableOn_uIoc ?_) ?_
       · -- Porting note: was `continuity`
         refine .mul continuous_const <| .mul (.div_const ?_ _) continuous_const
```

Comment:
All this block can be replaced with just `fun_prop`. Move it even to the `refine` above?

Reply:
Done, and unindented the refine.